src/common/common.cpp

```diff
@@ -6,6 +6,12 @@
 #include <sddl.h>
 #include "version.h"
 
+namespace localized_strings
+{
+  const wchar_t LAST_ERROR_FORMAT_STRING[] = L"%s failed with error %d: %s";
+  const wchar_t LAST_ERROR_TITLE_STRING[] = L"Error";
```

Comment:
Can you move these strings to the .rc file?

Reply:
`common` is a static library. so it's not possible. I'll try introducing a new `common_localization` DLL library or something...

Reply:
I've created a separate issue for it, since it involves too much logic changes - https://github.com/microsoft/PowerToys/issues/1311.